tests/integration/test_workflow_files.py

```diff
@@ -266,6 +266,7 @@ def _unlink(*args):
             '\nmocked-os-error'
         ),
     )) is remove_failed
+    monkeypatch.undo()
```

Comment:
the `_unlink` function at 221 seemed to leak into the teardowns use of `rmtree` causing the clean-up to fail.